docs/adrs/ADR-1.md

```diff
@@ -13,29 +12,97 @@ Deepansh
+    | percentage | Marks scored by user / total marks possible |
+    | section | :question: Will these questions also have subject wise sections like P/C/M/B? |
+    | max_marks_possible | total graded questions * score for correct answer |
+    | user_data_validated | does the user id exist in auth layer logs |
+    | avg_test_score | avg of scores of users in a particular session |
+    | highest_test_score | highest of scores of users in a particular session |
+    | rank | rank of student among other students in a particular session |
+    | percentile | percentile of student among other students in a particular session |
+    | num_test_takers | number of unique users who attempted the quiz |
+    | session_id | unique session id |
+    | start_date | start date of session |
+    | start_time | start time of session |
+    | end_date | end date of session |
+    | end_time | end time of session |
+    | infinite_session | :question: Should we add this? if yes, then this needs to go into firestore sessions DB as well |
+    | test_name | Will be taken as it is from firestore session DB |
+    | test_type | Will be taken as it is from quiz metadata `quiz_type` field from mongoDB - will be `homework` |
+    | cms_test_id | Will be taken as it is from quiz metadata `source_id` field from mongoDB |
+    | total_questions | Will be taken as it is from quiz metadata `num_graded_questions` field from mongoDB |
+    | group | Will be taken as it is from firestore session DB |
+    | batch | Will be taken as it is from firestore session DB |
+    | af_rank | Will be `null` for now, will be implemented using a different ADR |
+    | has_quiz_ended | Will be added |
+    | platform | Will be `quizengine` |
+    | inserted_at_timestamp | when was a record inserted into the table |
+    | ... new field? ... | ... what all new fields are needed? ... |
+
+  6. A table like the above one needs to be created if we decide on saving question level analysis as well for homework quizzes.
+  7. Should the existing ETL Flow code for assessments be reused or a new code file be created for this?
+
 ## Solution(s)
 > If a solution has been discussed then that should be discussed here. If multiple solutions exist, and their discussion is not feasible/practical on discord, then discuss these potential alternatives and their impact.
 > What alternatives are being considered, their benefits, their costs (team resources, money, time frames), mitigations for any drawbacks. The team will collectively take a decision in the Pull Request itself. The agreed upon decision then should be documented in the PR.
 
+Proposed solutions for the questions above:
+  1. After writing this, I think this should be stored in the same `test_responses` table. As the data will be almost similar to the assessments data. The only difference being the `test_type` being `homework`.
```

Comment:
Disagree. The decision of same table or not should be primarily based on the final use case (like I mentioned above). It can be exactly the same flow (since they are so similar) with the table name as just a parameter based on the session type